portfolio/src/main/webapp/style.css

```diff
@@ -1,21 +1,148 @@
 :root {
-  --general-color: rgba(236, 233, 227, 1);
+  --general-color: white;
 }
 
 body {
-  background-color: #292929;
+  background-color: white;
+}
+
+#page_container {
+  margin-right: auto;
+  margin-left: auto; 
+}
+
+#page_wrapper {
+  display: flex;
+  border: 1px solid darkgrey;
+  border-radius: 8px;
+  width: 100%;
 }
```

Comment:
Following the CSS guidelines, it is recommended to order properties in alphabetical order.